app/src/main/java/com/example/daznsportslist/Adapter_RV.kt

```diff
@@ -0,0 +1,31 @@
+package com.example.daznsportslist
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView.Adapter
+import androidx.recyclerview.widget.RecyclerView.ViewHolder
+
+class Adapter_RV(val playerList: List<String>) : Adapter<Adapter_RV.rv_ViewHolder>() {
+
+
+    class rv_ViewHolder(itemView: View) : ViewHolder(itemView) {
+        val playerName = itemView.findViewById<TextView>(R.id.txtTitle)
```

Comment:
Use view Binding instead of findviewById

Reply:
done